docs/wd_design.md

```diff
@@ -612,3 +674,86 @@ interface that is defined by vendor driver.
 When application doesn't want to access hardware accelerator, vendor driver 
 could invokes *wd_release_ctx()* to release the hardware.
 
+
+### Example with environment variable
+
+There's an example of compression and decompression with environment variable. 
```

Comment:
can we move this sample code to a directory, like in uadk/samples

Reply:
fixed